.gitignore

```diff
@@ -69,4 +69,9 @@ tests/recorded_test_cases/ObjectClassification/cube_objects_raw.npy
 .idea/
 
 
+# vim undo/swp file
+*.un~
+*.swp
 
+# pytest output
+tests/test_tracking_project.ilp
```

Comment:
I would rather change the test in such a way that the file is created outside (`tempfile`...) than adding this exclusion here.
But you don't have to do it, I can add this when I have some time.

Reply:
OK, I will remove it.